modules/genomic_browser/README.md

```diff
@@ -1,35 +1,35 @@
 # Genomic Module
 
-## Methylation tab
+## Purpose
 
-To use the methylation tab, the genomic_cpg_annotaion table should be filled with data from the Illumina HumanMethylation450k probe annotations file. This file can be found at ftp://webdata2:webdata2@ussd-ftp.illumina.com/downloads/ProductFiles/HumanMethylation450/HumanMethylation450_15017482_v1-2.csv. We provide a script,  `modules/genomic_browser/tools/HumanMethylation450k_annotations_to_sql` to transfer the csv file from Illumina's FTP to into a mysql transaction file. The output of the tool can be piped to mysql. The process should take between 5 to 10 minutes. 
+Unknown
```

Comment:
please be more descriptive

Reply:
I can't, I don't know anywhere that it's currently described and it can't be derived from the code. I just reformatted the information that was already in the old README.

Reply:
Try the help text ?

Reply:
I added something based on the first sentence in the help text